anax-in-container/horizon-container

```diff
@@ -379,7 +379,7 @@ stop() {
 	# Stop the anax container
 	echo "Unregistering the node, then stopping/removing the horizon container (this may take a minute)..."
```

Comment:
Maybe update this message so that people know the node will be removed

Reply:
thanks... updated